src/containers/Storage/Storage.tsx

```diff
@@ -28,6 +31,32 @@ import {defaultSortNode, getDefaultSortGroup} from './utils';
 
 import './Storage.scss';
 
+export function useStorageSort<
```

Comment:
Maybe move it to hooks folder? Not to expand component file with hooks. Large files with all-in-one have worse readability